.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '[0-9].[0-9]+'
+  pull_request:
+
+jobs:
+  test:
+    name: "PHPUnit"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.2', '7.3', '7.4', '8.0']
+        include:
+          - php-version: '7.3'
```

Comment:
why not 7.2 for the lowest version build?

Reply:
It's a mistake; fixed.